sections/header.liquid

```diff
@@ -170,63 +170,71 @@
                   <ul class="list list-social list-unstyled" role="list">
                     {%- if settings.social_twitter_link != blank -%}
                       <li class="list-social__item">
-                        <a href="{{ settings.social_twitter_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+                        <a href="{{ settings.social_twitter_link }}" class="list-social__link link">
                           {%- render 'icon-twitter' -%}
                           <span class="visually-hidden">{{ 'general.social.links.twitter' | t }}</span>
                         </a>
                       </li>
                     {%- endif -%}
                     {%- if settings.social_facebook_link != blank -%}
                       <li class="list-social__item">
-                        <a href="{{ settings.social_facebook_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+                        <a href="{{ settings.social_facebook_link }}" class="list-social__link link">
                           {%- render 'icon-facebook' -%}
                           <span class="visually-hidden">{{ 'general.social.links.facebook' | t }}</span>
                         </a>
                       </li>
                     {%- endif -%}
                     {%- if settings.social_pinterest_link != blank -%}
                       <li class="list-social__item">
-                        <a href="{{ settings.social_pinterest_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+                        <a href="{{ settings.social_pinterest_link }}" class="list-social__link link">
                           {%- render 'icon-pinterest' -%}
                           <span class="visually-hidden">{{ 'general.social.links.pinterest' | t }}</span>
                         </a>
                       </li>
                     {%- endif -%}
                     {%- if settings.social_instagram_link != blank -%}
                       <li class="list-social__item">
-                        <a href="{{ settings.social_instagram_link }}" class="link list-social__link" aria-describedby="a11y-external-message">
+                        <a href="{{ settings.social_instagram_link }}" class="list-social__link link">
                           {%- render 'icon-instagram' -%}
                           <span class="visually-hidden">{{ 'general.social.links.instagram' | t }}</span>
                         </a>
                       </li>
                     {%- endif -%}
+                    {%- if settings.social_tiktok_link != blank -%}
+                      <li class="list-social__item">
+                        <a href="{{ settings.social_tiktok_link }}" class="list-social__link link" aria-describedby="a11y-external-message">
```

Comment:
```suggestion
                        <a href="{{ settings.social_tiktok_link }}" class="list-social__link link">
```

Reply:
Just missing this one and I think it's good 👌

Reply:
hmm, not sure how that one got by. Good spot